$MirOS: ports/textproc/scrollkeeper/patches/patch-configure_in,v 1.1 2005/12/29 23:20:23 bsiegert Exp $
--- configure.in.orig	Sat Dec  6 02:54:15 2003
+++ configure.in	Sun Mar 16 16:33:04 2008
@@ -47,7 +47,7 @@ dnl ====================================
 dnl ====================================
 dnl = Begin checks for library functions.
 dnl ====================================
-AC_CHECK_FUNCS(getcwd getwd putenv strdup)
+AC_CHECK_FUNCS(getcwd getwd putenv strdup strndup)
 dnl ====================================
 dnl = End checks for library functions.
 dnl ====================================
@@ -235,7 +235,7 @@ AC_ARG_WITH(partial-db-dir,
       ;;
    auto)
       case "$host_os" in
-      freebsd* | netbsd* | openbsd*)
+      freebsd* | netbsd* | openbsd* | mirbsd*)
    	SCROLLKEEPER_PARTIAL_DB_DIR=db/scrollkeeper
    	;;
       *)
